SUPPORT.md

```diff
@@ -7,4 +7,6 @@ issues](https://github.com/microsoft/vscode-dotnettools/issues) before filing ne
 
 ## Microsoft Support Policy  
 
-Support for this extension is primarily made available via the GitHub Issues noted here. Technical Support may also be available from Microsoft's Customer Support Services (CSS). If you are a Premier or Unified Support customer, you may reach out to your account manager for further assistance. Otherwise, you may visit Microsoft's [Support for Business site](https://support.serviceshub.microsoft.com/supportforbusiness/create) to open a new support case for this extension.
\ No newline at end of file
+Support for this extension is primarily made available via the GitHub Issues noted here. Technical Support may also be available from Microsoft's Customer Support Services (CSS). If you are a Premier or Unified Support customer, you may reach out to your account manager for further assistance. Otherwise, you may visit Microsoft's [Support for Business site](https://support.serviceshub.microsoft.com/supportforbusiness/create) to open a new support case for this extension.
+
+Note: The .NET MAUI Extension for VS Code is not supported by Microsoft's Customer Support Services at this time. Issues related to the .NET MAUI Extension for VS Code should be reported from VS Code's 'Report Issue' commabnd (available from the Help menu).

```

Comment:
Feedback:
- don't capitalize 'Extension' (2 instances)
- typo for word 'command'

Reply:
Done. Thanks.